lib/clerk/railtie.rb

```diff
@@ -5,6 +5,12 @@ module Clerk
   class Railtie < Rails::Railtie
     initializer 'clerk.ar_extensions' do |app|
       ActiveRecord::Base.extend Clerk
+
+      #initialize configuration object
+      Clerk.configure do |config|
+        config.silence_warnings = Rails.env.test?
```

Comment:
I'm not a 100% sure about defaulting to true for test environment?

Let's go with it for now - might change this in the future; I guess it is easy enough to enable in spec helper